src/components/CommentCard.jsx

```diff
@@ -0,0 +1,26 @@
+const CommentCard = ({ comments }) => {
+  return (
+    <ul className="comment-card">
+      {comments.map((comment) => {
```

Comment:
I'd think about here what would happen if there were no commnets yet for an article.
You'd probably get an error here as you cannot map an empty array.